module-check/src/test/clojure/clojure/core/typed/test/template.clj

```diff
@@ -0,0 +1,23 @@
+(ns clojure.core.typed.test.template
+  (:require 	[clojure.core.typed :as t]
+                [clojure.template :as template] 
+		[clojure.xml :as xml]
+		[clojure.zip :as zip]
+		[clojure.test :refer :all]                
+		[clojure.core.typed.test.test-utils :refer :all]))
+
+(deftest applyTemplate-test
+  (is-tc-e   (clojure.template/apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))t/Any) 
```

Comment:
You need to put a space between a symbol and a list in t/Any. But here it isn't doing anything because everything is of type Any, so you can delete the expected type.

The correct syntax for requiring vars in other namespaces is:

```clojure
(is-tc-e   (apply-template '[a b c d e] '[d a b e c e b a d] '(1 2 3 4 5))
              :requires [[clojure.template :refer [apply-template]]])
```

If this is slightly wrong, please look again at the core.async tests.